terra-jupyter-base/Dockerfile

```diff
@@ -105,36 +89,21 @@ RUN useradd -m -s /bin/bash -N -u $WELDER_UID $WELDER_USER
 # ensure this matches c.NotebookApp.port in jupyter_notebook_config.py
 ENV JUPYTER_PORT 8000
 ENV JUPYTER_HOME /etc/jupyter
-ENV USER_INSTALL_DIR /home/jupyter-user/notebooks/packages
+ENV USER_INSTALL_DIR /home/jupyter/notebooks/packages
 
 # Prepend existing PYTHONPATH if it's not empty; otherwise prepend it
 ENV PYTHONPATH ${PYTHONPATH:+$PYTHONPATH:}$JUPYTER_HOME/custom:$USER_INSTALL_DIR
 ENV PATH ${PATH:+$PATH:}$USER_INSTALL_DIR/bin
 
-# # Python3, kernel, and pip
-RUN apt update \
-   && apt install -yq software-properties-common \
-   && add-apt-repository ppa:deadsnakes/ppa \
-   && apt install -yq python3.7-dev \
-   && update-alternatives --install /usr/local/bin/python3 python3 /usr/bin/python3.7 1 \
-   && apt-get -yq install python3-pip \
-   && /usr/local/bin/python3 -m pip install --upgrade pip
-
 RUN pip3 -V \
  # When we upgraded from jupyter 5.7.8 to 6.1.1, we broke terminal button on terra-ui.
  # Hence, make sure to manually test out "launch terminal" button (the button in the green bar next to start and stop buttons)
  # to make sure we don't accidentally break it every time we upgrade notebook version until we figure out an automation test for this
- && pip3 install notebook==6.1.1 \
- && pip3 install jupyter==1.0.0 \
  && pip3 install python-datauri \
- && pip3 install jupyterlab==0.35.4 \
- && pip3 install cookiecutter \
  && pip3 install jupyter_contrib_nbextensions \
  && pip3 install jupyter_nbextensions_configurator \
  # for jupyter_delocalize.py and jupyter_notebook_config.py
- && pip3 install tornado==5.1.1 \
  && pip3 install requests \
- && pip3 install six==1.15.0 \
  && pip3 install firecloud==0.16.25 \
```

Comment:
Might now be a good time to remove the couple Python package version pins in this Dockerfile?

Reply:
yea, maybe...I'll check to see what other ppl think